app/controllers/AMIable.scala

```diff
@@ -42,9 +42,10 @@ class AMIable @Inject()(override val amiableConfigProvider: AmiableConfigProvide
     attempt {
       for {
         amis <- Prism.getAMIs()
-        ami = amis.find(_.imageId == imageId)
-        amiWithUpgrade = ami.map(Recommendations.amiWithUpgrade(agents.allAmis))
-      } yield Ok(views.html.ami(amiWithUpgrade))
+        maybeAmi = amis.find(_.imageId == imageId)
+        instances <- Prism.imageUsage(maybeAmi)
+        amiWithUpgrade = maybeAmi.map(Recommendations.amiWithUpgrade(agents.allAmis))
+      } yield Ok(views.html.ami(amiWithUpgrade, instances))
```

Comment:
We also need to take `launch configurations` into account. An AMI could be for example used in a `launch configuration` with an ASG that has currently 0 instances.
This might be added in a following pull request though